frontend/packages/console-app/src/utils/isMultiClusterEnabled.ts

```diff
@@ -0,0 +1,11 @@
+const isMultiClusterEnabled = (): boolean => {
+  let isAvailable = false;
+  // Mock implementation. needs update
+  const mchFlagData = window.SERVER_FLAGS.clusters;
+  if (mchFlagData) {
+    isAvailable = mchFlagData.length > 1; // to-do: update logic/flag when finalised
+  }
+  return isAvailable;
+};
```

Comment:
nit, this could be simplified with optional chaining:
```suggestion
const isMultiClusterEnabled = (): boolean => window.SERVER_FLAGS.clusters?.length > 1;
```

Reply:
updated